packages/form-js-editor/test/spec/FormEditor.spec.js

```diff
@@ -992,7 +992,7 @@ describe('FormEditor', function() {
     });
 
 
-    it('should move row', async function() {
+    it.skip('should move row', async function() {

```

Comment:
This becomes laggy due to the absolute positioning now, I think. That's always hard to test, so I got your rationale about this particular test.

In general, I think the dragging tests make sense, as the behavior is predictable.

Reply:
Can we at least add an comment above the test why we skip it and maybe a note that we could solve it with proper integration testing? Or should we remove it entirely?

Reply:
We can try to move this test o Playwright when I create the setup